website/pages/en/querying/querying-the-hosted-service.mdx

```diff
@@ -19,9 +19,9 @@ This query lists all the counters our mapping has created. Since we only create
 }
 ```
 
-## Using The Hosted Service
+## Using The hosted service
```

Comment:
Haha, maybe this one shouldn't be changed, or "The" should be all lowercase too?